core/src/test/scala/com/broadinstitute/dsp/DbTestHelper.scala

```diff
@@ -29,7 +29,7 @@ object DbTestHelper {
   ): Resource[IO, HikariTransactor[IO]] =
     for {
       xa <- DbTransactor.init[IO](databaseConfig)
-      _ <- Resource.eval(truncateTables(xa))
+      _ <- Resource.make(IO.unit)(_ => truncateTables(xa))
```

Comment:
This is a small unrelated fix around unit tests.

Today, we can't run lots of the DB related unit tests because when they're run in bulk, they will fail for some reason. And I think this bug is one of the reasons (my own fault)...

So with this change, I'm able to remove `DbTest` tag all unit tests in `zombieMonitor`...I'll see if I can remove more if I have time